Composer/packages/extensions/visual-designer/src/index.tsx

```diff
@@ -62,8 +62,7 @@ const VisualDesigner: React.FC<VisualDesignerProps> = ({
 
   const focusedId = Array.isArray(focusedActions) && focusedActions[0] ? focusedActions[0] : '';
 
-  // NOTE: avoid re-render. https://reactjs.org/docs/context.html#caveats
-  const [context, setContext] = useState({
```

Comment:
Notes here: the usage of `useState` here was correct before we hoisting the `focusedId` to Shell but not necessary any more after our hoisting PRs.